test/multinomial.jl

```diff
@@ -64,6 +64,18 @@ end
 # test type stability of logpdf
 @test typeof(logpdf(convert(Multinomial{Float32}, d), x1)) == Float32
 
+# test degenerate cases of logpdf
+d1 = Multinomial(1, [0.5, 0.5, 0.0])
+d2 = Multinomial(0, [0.5, 0.5, 0.0])
+x2 = [1, 0, 0]
+x3 = [0, 0, 1]
+x4 = [1, 0, 1]
+
+@test logpdf(d1, x2) == log(0.5)
```

Comment:
For float equality it's better to check `@test logpdf(d1, x2) ≈ log(0.5)`. That ensures there's a sensible tolerance. (In case you aren't familiar, `≈`, which can be completed at the REPL as `\approx<tab>`, calls the function `isapprox`.)